website/package.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Where is the package lock ?

Reply:
not sure if we need to commit it (there are diff opinions, same as terraform lock), but added